Modules/Devices/src/Devices.Application/Identities/Commands/StartDeletionProcessAsUser/StartDeletionProcessAsUserResponse.cs

```diff
@@ -17,9 +18,7 @@ public StartDeletionProcessResponse(IdentityDeletionProcess deletionProcess)
     public string Id { get; set; }
     public DeletionProcessStatus Status { get; set; }
     public DateTime CreatedAt { get; set; }
-
     public DateTime ApprovedAt { get; set; }
     public DeviceId ApprovedByDevice { get; set; }
-
     public DateTime GracePeriodEndsAt { get; set; }
```

Comment:
The blank lines have been there for a reason. They were separating the properties based on the phases they are first populated by.